docs/development.rst

```diff
@@ -143,36 +143,43 @@ like Sphinx 1.8, HTML4, or required support for IE11.
 2.0.0
 ~~~~~
 
-:Planned release date: 2022 Q1
+:Planned release date: 2023 Q3
 
-This release will mark the beginning of a new round of feature development, as
-well as a number of backward incompatible changes and deprecations.
+This release will drop support for old Python and Sphinx versions.
 
-Of note, the following backwards incompatible changes are planned for this
+Note the following backwards incompatible changes are planned for this
 release:
 
-Sphinx 1.x, Sphinx 2.x, and Docutils 0.16 will not be tested
+Python < 3.8, Sphinx <5 and Docutils < 0.18 will not be tested
```

Comment:
This doesn't quite match what we talked about, though we also only talked about *what we'd support* and not what we'd *test against*.

But where we are testing Sphinx 5, we should look to the Sphinx 5 dependencies to dictate our support:

https://github.com/sphinx-doc/sphinx/blob/953002e6261bdd9314ee0a3314aae19479a88c7e/setup.py#L10-L31

So, Python 3.6 and above and docutils 0.14 and above.

However, we did not talk about what we would *test* against. This can certainly be different than what we want to support. `Python>=3.8, docutils>=0.18` would be valid if we say we test the last two versions of Sphinx, but seems odd to say we test against a different set of dependencies than we support too.